quickwit-cli/tests/cli.rs

```diff
@@ -272,9 +274,12 @@ async fn test_cmd_dry_run_delete_on_s3_localstack() -> Result<()> {
 }
 
 #[tokio::test]
+#[serial]
```

Comment:
why do you need serial ?

Reply:
the tests have a dependency currently since they both start a webserver

Reply:
indeed!